content/articles/core-concepts-of-django-to-master/index.md

```diff
@@ -0,0 +1,166 @@
+<!-- Author Biography -->
+Name: ARAFAT OLAWUMI OLAYIWOLA
+I am Software and Associate Cloud Engineer who is passionate about the over-whelming technologies out there. I develop a scalable and user friendly applications that run on all platforms and responsive. I love Django like my girlfriend. Undergraduate of Computer Engineering here in Nigeria.
+
+<!-- My Picture -->
+![Arafat Olayiwola](/core-concepts-of-django-for-mastering/Arafat-Olayiwola.jpg)
+
+![django-img](/core-concepts-of-django-for-mastering/django-image.png)
+
+<!-- INTRODUCTION -->
+Are you  a beginner or an intermediate of Django? Join me to explore more on this superb and sweet framework used to setup web apps by connecting to server. You don't want to miss this!!
+
+### Keys Take Away
+At the end of this article, you will be able to do the following;
+1. Setup Django project and configure it
+2. Have core understanding of how routers work
+3. Know what we mean by MVT in Django world
+4. Understand Django Admin Interface
+5. Be able to authenticate users 
+6. Know what is relational database management system
+
+Let's Begin!!!!
+
+What is Django Framework?
+
+"Django is a free, open source web framework written in the Python programming language". A “web framework” is software that abstracts away many of the common challenges related to building a website, such as connecting to a database, handling security, user accounts and much more. Django uses a python package called pip and this come in handy having installed Python. 
```

Comment:
Tip: 
If you are quoting this definition from another source, it is important to let the user know by listing this source and adding quotes around the definition.